package.json

```diff
@@ -27,13 +27,13 @@
     "rdf-parser-microdata": "^0.3.0",
     "rdf-parser-n3": "^0.3.0",
     "rdf-parser-rdfxml": "^0.3.0",
+    "rdf-ext": "^0.3.0",
```

Comment:
It's not about attaching the parsers and serializers to the default rdf object. The rdf.Parsers and rdf.Serializers classes are used if a new instance of formats is created (`var formats = require('rdf-formats-common')()`). See line [#20](https://github.com/rdf-ext/rdf-formats-common/blob/master/index.js#L20).